zerver/models.py

```diff
@@ -536,6 +536,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     MAX_NAME_LENGTH = 100
     API_KEY_LENGTH = 32
     NAME_INVALID_CHARS = ['*', '`', '>', '"', '@']
+    INVALID_FULL_NAME_RE = r"([^a-zA-Z ])"
```

Comment:
Since we support unicode characters in names, I don't think we can do something simple like this.